spec/requests/providers/means/identify_types_of_incomes_controller_spec.rb

```diff
@@ -2,8 +2,20 @@
 
 RSpec.describe Providers::Means::IdentifyTypesOfIncomesController do
   let(:legal_aid_application) { create(:legal_aid_application) }
-  let!(:outgoing_types) { create_list(:transaction_type, 3, :debit_with_standard_name) }
-  let!(:income_types) { create_list(:transaction_type, 3, :credit_with_standard_name) }
+  let!(:outgoing_types) do
+    [
+      create(:transaction_type, :rent_or_mortgage),
+      create(:transaction_type, :maintenance_out),
+      create(:transaction_type, :child_care),
+    ]
+  end
+  let!(:income_types) do
+    [
+      create(:transaction_type, :benefits),
+      create(:transaction_type, :pension),
+      create(:transaction_type, :maintenance_in),
+    ]
+  end
```

Comment:
nit - linebreak

Reply:
Personally I'd like to see a request spec here as well. bit belt and braces with the feature test but prefer to have to much

```
    context "when benefits selected" do
      let(:transaction_type_ids) { [benefits.id] }
      let(:benefits) { create(:transaction_type, :benefits) }

      before do
        create(:transaction_type, :excluded_benefits, parent_id: benefits.id)
      end

      it "adds excluded_benefits transaction type as well as benefits" do
        expect { request }.to change(LegalAidApplicationTransactionType, :count).by(2)
        expect(legal_aid_application.reload.transaction_types.pluck(:name)).to match_array(%w[benefits excluded_benefits])
      end
    end
```